### PR TITLE
PBM-539 Updated install/upgrade docs with installing fro pbm repo

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -44,14 +44,14 @@ Prerequisites
 ================================================================================
 
 It is recommended to install |pbm| from official |percona| repositories by using
-the |percona-release| utility.
+the |percona-release| utility. Starting from v1.2.1, |pbm| packages are stored in the *pbm* repository.
```
Comment (EvgeniyPatlan) [inline doc/source/installation.rst:47]:
```suggestion
the |percona-release| utility. Starting from v1.3.0, |pbm| packages are stored in the *pbm* repository.
```

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: doc/source/upgrading.rst [hunk 2]
```diff
@@ -22,7 +22,8 @@ You can upgrade |PBM| to the **latest version** or to a **specific version**. Si
 .. rubric:: Important notes
  
 1. Backward compatibility between data backup and restore is supported for upgrades within one major version only (e.g. from 1.1.x to 1.2.y). When you upgrade |PBM| over several major versions (e.g. from 1.0.x to 1.2.y), we recommend to make a backup right after the upgrade.
-2. Upgrade |PBM| on all nodes where it is installed.
+2. Starting from version 1.2.1, |pbm| packages are stored in the *pbm* repository and the *tools* repository for backward compatibility. To upgrade to the version lower than 1.2.1, enable the *tools* repository.
```
Comment (EvgeniyPatlan) [inline doc/source/upgrading.rst:25]:
```suggestion
2. Starting from version 1.3.0, |pbm| packages are stored in the *pbm* repository and the *tools* repository for backward compatibility. To upgrade to the version lower than 1.3.0, enable the *tools* repository.
```

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]
1.3.0

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by akira-kurogane — APPROVED** [review]
LGTM

**Pull request merged**